File: Dart/1005.dart
import 'dart:io';

void main() {
  double x = double.parse(stdin.readLineSync());
  double y = double.parse(stdin.readLineSync());
  x = x * 3.5;
  y = y * 7.5;
  double media = (x + y) / 11;
  print("MEDIA = " + media.toStringAsFixed(5));
}

File: JavaScript/1005.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

let a = parseFloat(lines.shift());
let b = parseFloat(lines.shift());

console.log("MEDIA = " + ((a * 3.5 + b * 7.5) / 11).toFixed(5));